python/py-pytest-flake8/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pytest-flake8
+version             1.0.7
+platforms           darwin
+license             Restrictive/Distributable
```

Comment:
why is this not just `BSD`? that's what is advertised on PyPI and also listed in [upstream's setup.py](https://github.com/tholo/pytest-flake8/blob/9db176caba1738df680bf574619b7f0f2dd85ac4/setup.py#L15)

Reply:
Thanks fixed.